test/example_config.yaml

```diff
@@ -0,0 +1,39 @@
+###################################################
+#
+# k8s-cluster.yaml
```

Comment:
As a general comment on this file, so that we don't duplicate effort in understanding CORTX as a whole and eventual multiple documentation domains, would we be able to align the structure of this file with what is in solution.yaml in a more 100% compatible way, such that you could in theory do a `yq merge` operation and overlay these values onto the default `solution.yaml`?

Reply:
I'm open to doing that, but one of the goals is to make it simple to configure.

For cvgs I find it annoying and verbose to name each of the data devices, to specify the size, and to specify "type: ios" (I don't know what it does).

For nodes I find it annoying to use the keys "node1", "node2", rather than just list the set of hostnames.

Reply:
Regarding putting these in a "regression" directory, good idea!  I'm happy to do that.

Reply:
I have moved this content from `test` to `test/regression` as requested.